client/src/components/Donation/Donation.js

```diff
@@ -104,7 +104,10 @@ class DonationModal extends Component {
             <Modal.Body>
               <DonateForm
                 email={email}
+                /**
```

Comment:
Hey @Bouncey why do we need this in the other app again? This was breaking the page, so I got rid of it. If you can confirm I'll remove the code and comments as well